libs/test/Test/Golden.idr

```diff
@@ -154,7 +164,8 @@ options args = case args of
 
     mkOptions : String -> List String -> IO (Maybe Options)
     mkOptions exe rest
-      = do let Just (mfp, opts) = go rest Nothing (initOptions exe)
+      = do color <- (Just "DUMB" /=) <$> getEnv "TERM"
```

Comment:
We should probably add a function to ANSI to check whether a terminal
supports highlighting. We already have at least 2 occurrences of this
recipe in the codebase and if we need to update it at some point, chances
are we'll only update one version.